### DISCUSSION TITLE
Welcome to cli Discussions!

### DISCUSSION THREAD

**Discussion by a-b** [cloudfoundry/cli#2244] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DayronDLR** [comment]
When will CloudFoundry CLI for Mac arm be available?

**Comment by sangareddynalla** [comment]
Hi

I am trying to install **credhub cli** on the Apple M2  ( Mac OS Ventura 13.2.1)
The below command was run from the terminal and it is giving the error.

$ brew install credhub-cli

credhub-cli: The x86_64 architecture is required for this software.
Error: credhub-cli: An unsatisfied requirement failed this build.

Please help to fix this issue.
